### PR TITLE
`if`: parse nonblock consequents

### PR DESCRIPTION
**Pull request by wcjohnson**
Fixes lightscript/lightscript#21

- Parsing for nonblock consequents
- Update/add fixtures

### CODE REVIEW

File: test/fixtures/lightscript/if-expression/vanilla/actual.js [hunk 1]
```diff
@@ -0,0 +1 @@
+x = if(true) 1
```
Comment (rattrayalex) [inline test/fixtures/lightscript/if-expression/vanilla/actual.js]: let's call this `paren-expr-body` instead of `vanilla` (ditto elsewhere)

**Review by rattrayalex — COMMENTED** [review]
One naming change, otherwise LGTM! Thanks for fixing this.

**Comment by wcjohnson** [comment]
I have another fix to add to this pr before merging, re the new bug I just filed. Hold off a bit on merging please.

**Comment by wcjohnson** [comment]
@rattrayalex 

Never mind on that other fix: the problem is much deeper than I thought. Fixture is renamed as requested; you can merge this fix by itself.

**Comment by rattrayalex** [comment]
Yeah that makes sense. I think I'm going to need to spend some time with that one. Thanks!

**Pull request merged**